### ISSUE TITLE
Input/Output indicators

### ISSUE THREAD

**Issue by LoganJFisher** [JerryI/wolfram-js-frontend#207]
**Is your feature request related to a problem? Please describe.**
Right now, if I input 2+2, the next block will show 4, but there is no indication at a glance of what is an input and what is an output.

**Describe the solution you'd like**
Mathematica resolves this by indicating to the left of each block: "In[#]=" or "Out[#]=", which I believe is the most elegant way of resolving this issue.

**Additional context**
WLJS: ![ZBtDGWk](https://github.com/JerryI/wolfram-js-frontend/assets/63993409/05efd87c-b2b5-4ee5-9e4e-b3098937f6e4)
Mathematica: ![nXIUpO1](https://github.com/JerryI/wolfram-js-frontend/assets/63993409/7dc518ad-b315-43d9-9718-e7ed7eea7dca)

**Comment by munuxi** [comment]
Yes, we need it, and it's very useful when we want to use output result as our variables e.g. %234 in the calculation.

**Comment by JerryI** [comment]
<img width="526" alt="Screenshot 2024-06-24 at 11 01 05" src="https://github.com/JerryI/wolfram-js-frontend/assets/4111822/4ad5a37d-5c11-48af-a0ef-cbe466bd9410">

In principle, this is indicated by a gray bracket. But anyway, i will try to find a place in HTML layout for additional labels

**Comment by JerryI** [comment]
<img width="830" alt="Screenshot 2024-06-24 at 11 54 06" src="https://github.com/JerryI/wolfram-js-frontend/assets/4111822/187f8431-17eb-4a09-b511-a3f919051b8f">

We need UI designer ;D

**Comment by Enivex** [comment]
This is very useful to refer to previous output as well when experimenting.

**Comment by JerryI** [comment]
Referencing the previous output is possible via `%`, but only the last output is preserved (effective history length is 1).
